edsdme/scripts/utils.js

```diff
@@ -115,6 +126,18 @@ export function isMember() {
   return status === 'MEMBER';
 }
 
+export function partnerIsSignedIn() {
+  return getCookieValue('partner_data');
```

Comment:
@zagi25 I don't think we have this cookie if the user is signed in but is not a member of the portal.
We can check this with adobeIMS.isSignedInUser(), can you check if there are timing issues if we use this from adobeIMS?

Reply:
@Ben-Zahler Everyone who visits our pages and is logged in with IMS will get cookie, members will get cookie with information provided by `partnerservices` and non members will get cookie with fake information.